atomix/core/src/main/java/io/atomix/core/Atomix.java

```diff
@@ -109,6 +110,14 @@ protected Atomix(
         Executors.newScheduledThreadPool(
```

Comment:
:thought_balloon: Kind of out of scope, but...is this actually used anywhere? Because it looks to me like it isn't and we're just spawning random threads? :sweat_smile: